app-backend/package.json

```diff
@@ -5,23 +5,30 @@
 	"main": "index.js",
 	"scripts": {
 		"start": "nodemon ./src/index.js",
-		"test": "echo \"Error: no test specified\" && exit 1"
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"lint": "eslint ./ --fix"
 	},
 	"author": "Kirill Stepanchuk <kirill.steapnchuk.96@mail.ru>",
 	"license": "ISC",
 	"dependencies": {
-		"airbnb": "0.0.2",
 		"cookie-parser": "^1.4.5",
 		"cors": "^2.8.5",
 		"dotenv": "^8.2.0",
-		"eslint": "^7.25.0",
+		"eslint-config-airbnb": "^19.0.2",
```

Comment:
please, read about  dependencies and devDependencies and fix deps in your package.json